src/content/en/updates/2019/06/layoutNG.md

```diff
@@ -0,0 +1,328 @@
+When an element forming a block formatting context is sized next to floats, 
+the legacy layout engine would try to size the block a fixed number 
+of times before giving up. This approach led to unpredictable and unstable 
+behavior and didn't match other implementations. In LayoutNG all 
+floats are taken into account when sizing the block. 
+(See [Chromium bug #548033](https://crbug.com/548033).)
+
+Absolute and fixed positioning are now more compliant with W3C specifications 
+and better match the behavior in other browsers. The differences 
+between the two are most visible in two cases:
+
+- **Multi-line inline containing blocks** <br>If an absolutely positioned 
+containing block spanned multiple lines, the legacy engine might 
+incorrectly use only a subset of the lines to compute the containing block bounds.
+- **Vertical writing modes** <br>The legacy engine had many problems placing 
+out-of-flow elements in the default position in vertical writing modes. 
+See the next section for more details about improved writing mode support.
+
+## Right-to-left (RTL) languages and vertical writing modes
+LayoutNG was designed from the ground up to support vertical writing modes 
+and RTL languages, including bidirectional content.
+
+#### Bidirectional Text
+LayoutNG supports the most up-to-date bidirectional algorithm defined by 
+[The Unicode Standard](https://unicode.org/standard/standard.html). 
+Not only does this update fix various rendering errors, but it also includes 
+missing features such as paired bracket support 
+(See [Chromium bug #302469](https://crbug.com/302469).)
+
+#### Orthogonal Flows
+LayoutNG improves the accuracy of vertical flow layout, including, for example, 
+placement of absolutely positioned objects and sizing of orthogonal 
+flow boxes (especially when percent is used). Of the 1,258 tests in the 
+W3C test suites, **103 tests that failed in the old layout engine pass in LayoutNG.**
+
+#### Intrinsic Sizing
+Intrinsic sizes are now calculated correctly when a block contains children 
+in an orthogonal writing mode.
+
+### Text Layout & Line Breaking
+The legacy Chromium layout engine laid out text element-by-element and line-by-line. 
+This approach worked well in most cases but required a lot of e
+xtra complexity to support scripts and achieve good performance. It was also 
+prone to measurement inconsistencies, which led to subtle differences 
+in the sizing of size-to-content containers and their content or unnecessary line breaks.
+
+In LayoutNG, text is laid out at the paragraph level and then split into lines. 
+This allows for better performance, higher quality text rendering, 
+and more consistent line breaking. The most notable differences are detailed below.
+
+#### Joining across Element Boundaries
+In some scripts, certain characters can be visually joined when they're adjacent. 
+Check out this example from Arabic:
+
+[Add a two-up showing two characters separate, and then the same characters joined.]
```

Comment:
Is this a TODO?

Reply:
hehe nope

Reply:
i think so, and no idea where it came from..